src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,14 +209,21 @@ function toggleCollapse(e, pans) {
     var panels = pans || $('div.panel-collapse');
 
     if ($(e).html().trim().startsWith(expand)) {
-        isExpandingAll = true;
-        var i = 0;
-        for (var idx = 0; idx < panels.length; idx++) {
-            if ($(panels[idx]).attr('class').indexOf('in') === -1) {
-                setTimeout(showSingleCollapse, 50 * i, panels[idx]);
-                i++;
+        $('.ajax-response-auto').click();
+        var $autoLoadCount = $('.ajax_auto').click();
+        var $isAutoLoadingTriggered = $autoLoadCount.length !== 0;
+
+        if (!$isAutoLoadingTriggered) {
+            isExpandingAll = true;
+            var i = 0;
+            for (var idx = 0; idx < panels.length; idx++) {
+                if ($(panels[idx]).attr('class').indexOf('in') === -1) {
+                    setTimeout(showSingleCollapse, 50 * i, panels[idx]);
```

Comment:
I am suspicious of this method ```showSingleCollapse``` as it permanently adds a class of ```in``` and a style of ```height: auto;``` that might cause panels to behave oddly, can you check the function and if it does do all those stuff, please open a new issue and describe that this function should not be doing that